From 7a6e8dd824ab470619c90609969f05a4adefe082 Mon Sep 17 00:00:00 2001
From: Hao Bui <hao.bui.yg@renesas.com>
Date: Sun, 10 May 2020 01:52:28 +0700
Subject: [PATCH 084/433] mmc: renesas_sdhi: Add r8a7742 support

Add support for r8a7742 SoC (RZ/G1H). Renesas RZ/G1H SDHI is identical
to the R-Car Gen2 family.

Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 drivers/mmc/host/renesas_sdhi_sys_dmac.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mmc/host/renesas_sdhi_sys_dmac.c b/drivers/mmc/host/renesas_sdhi_sys_dmac.c
index d192788..02ec13e 100644
--- a/drivers/mmc/host/renesas_sdhi_sys_dmac.c
+++ b/drivers/mmc/host/renesas_sdhi_sys_dmac.c
@@ -84,6 +84,7 @@ static const struct of_device_id renesas_sdhi_sys_dmac_of_match[] = {
 	{ .compatible = "renesas,sdhi-r8a7744", .data = &of_rcar_gen2_compatible, },
 	{ .compatible = "renesas,sdhi-r8a7745", .data = &of_rcar_gen2_compatible, },
 	{ .compatible = "renesas,sdhi-r8a77470", .data = &of_rcar_gen2_compatible,},
+	{ .compatible = "renesas,sdhi-r8a7742", .data = &of_rcar_gen2_compatible,},
 	{ .compatible = "renesas,sdhi-r8a7790", .data = &of_rcar_gen2_compatible, },
 	{ .compatible = "renesas,sdhi-r8a7791", .data = &of_rcar_gen2_compatible, },
 	{ .compatible = "renesas,sdhi-r8a7792", .data = &of_rcar_gen2_compatible, },
-- 
2.7.4

